src/workerd/api/kv-test.js

```diff
@@ -150,18 +172,21 @@ export default {
 
     // // get bulk json but it is not json - throws error
     await assert.rejects(env.KV.get(['key-not-json', 'key2'], 'json'), {
-      message: 'KV GET_BULK failed: 500 Internal Server Error',
+      message:
+        'KV GET_BULK failed: 400 At least one of the requested keys corresponds to a non-json value',
     });
 
     // // requested type is invalid for bulk get
     await assert.rejects(env.KV.get(['key-not-json', 'key2'], 'arrayBuffer'), {
-      message: 'KV GET_BULK failed: 500 Internal Server Error',
+      message:
+        'KV GET_BULK failed: 400 "arrayBuffer" is not a valid type. Use "json" or "text"',
     });
 
     await assert.rejects(
       env.KV.get(['key-not-json', 'key2'], { type: 'banana' }),
       {
-        message: 'KV GET_BULK failed: 500 Internal Server Error',
+        message:
+          'KV GET_BULK failed: 500 "banana" is not a valid type. Use "json" or "text"',
```

Comment:
Nit: This needs to be 400 instead of 500, that should get the test (and thus CI checks) to pass.